tokenomia setup.md

```diff
@@ -0,0 +1,38 @@
+ # Tokenomia setup
+
+ ### to instantiate the node: ./testdata.json
+
+ docker run --rm \
+  -e NETWORK=testnet \
+  -v "$PWD"/.node/socket:/ipc \
+  -v "$PWD"/.node/data:/data \
+  inputoutput/cardano-node:1.34.0
+
+ after the node has synced (confirm by comparing slot to most recent Tx on cardanoscan), open a new terminal and run the following command to give the user permission to write in order to set an environment variable:
+
+  sudo chown $(whoami) "$PWD"/.node/socket/node.socket
+
+Then we need to set 2 env variables:
```

Comment:
Might want to work on your counting skills here 😅 (or make it more clear that you pick one of the 2 token paths)